src/Raven.Server/Documents/Handlers/AI/Agents/ConversationDocument.cs

```diff
@@ -315,6 +324,34 @@ public static List<BlittableJsonReaderObject> GenerateTools(JsonOperationContext
             tools.Add(context.ReadObject(tool, "tool"));
         }
 
+        foreach(var subAgent in configuration.SubAgents ?? [])
+        {
+            AiAgentConfiguration subAgentConfiguration = handler.GetAiAgentConfiguration(subAgent.Identifier);
+            var argsSampleObject = new DynamicJsonValue();
+            foreach (AiAgentParameter parameter in subAgentConfiguration.Parameters ?? [])
```

Comment:
what about the parameters from the parent?
do we want to pass them here as well?